Framework/DataHandling/src/LoadInstrument.cpp

```diff
@@ -95,74 +97,103 @@ void LoadInstrument::init() {
  */
 void LoadInstrument::exec() {
   // Get the input workspace
-  m_workspace = getProperty("Workspace");
-  m_filename = getPropertyValue("Filename");
-  m_instName = getPropertyValue("InstrumentName");
+  boost::shared_ptr<API::MatrixWorkspace> ws = getProperty("Workspace");
+  std::string filename = getPropertyValue("Filename");
+  std::string instname = getPropertyValue("InstrumentName");
 
-  // We will parse the XML using the InstrumentDefinitionParser
+  // For IDF files, we will parse the XML using the InstrumentDefinitionParser
   InstrumentDefinitionParser parser;
```

Comment:
Not sure this `InstrumentDefinitionParser` at this scope now does anything? Can it be removed?